background.js

```diff
@@ -70,10 +71,19 @@ $(document).ready(function () {
     }
     setTimeout(checkOnline, 1000);
   }
+  
+  /*setting intial status icon
```

Comment:
shouldnt you remove this if this is not needed

Reply:
Done!